install/manifests.yaml

```diff
@@ -916,6 +923,25 @@ spec:
                       description: Whether this identity config should generate individual
                         observability metrics
                       type: boolean
+                    mtls:
+                      properties:
+                        allNamespaces:
+                          default: false
+                          description: Whether Authorino should look for TLS secrets
+                            in all namespaces or only in the same namespace of the
+                            AuthConfig. Enabling this option in namespaced Authorino
```

Comment:
`in the same namespace as the AuthConfig` or `in the namespace of the AuthConfig` ?

Reply:
Fixed. Thanks.